packages/core/src/enums/MetadataModules.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Cornerstone Core events
+ */
+enum MetadataModules {
+  CINE_MODULE = 'cineModule',
+  IMAGE_URL_MODULE = 'imageUrlModule',
```

Comment:
I saw the implemntation of this, so our naming convention is `imageId = schema + imageURI``, can you name this `IMAGE_URI_MODULE` ?

Reply:
Hmmm looking again at this, I'm wrong. This seems to be a lot specific to video, can we call this then `VIDEO_URL_MODULE`?

Reply:
There is currently some specific to video, but it is a general image url module, neither URI nor video specific.
The intent is to use it for things like thumbnail URLs, rendered endpoints etc, all of which are URL based, not just URI, and those are not necessarily video related.

Reply:
Some of the non-video stuff is things like PDF viewing - which happens to use the same rendered endpoint, I just haven't implemented it yet.